### ISSUE TITLE
Avocode crashes if I try to launch it again while it's already running

### ISSUE THREAD

**Issue by madve2** [avocode/avocode#2172]
If I launch Avocode when it's already running (it easily goes unnoticed in a multi-monitor setup), the already running instance gets focus (as it should), but the Unexpected error window pops up, with the message you see below.

---
* **Version**: 3.5.3
* **Web/Desktop**: Desktop
* **Platform**: win32
* **Avocode UID**: 214267
---
```
TypeError: t.indexOf is not a function
    at EventEmitter._handleAnotherInstance (file:///C:/Users/kis-n/AppData/Local/avocode/app-3.5.3/resources/app.asar/dist/avocode.min.js:1:3090137)
    at EventEmitter.emit (events.js:182:13)
```

**Comment by crashmajstr** [comment]
Hi @madve2 
thanks for reporting this. I've just tried to reproduce it and I've experienced the same issue. So I'll hand it out to our dev team and reach out to you with the solution asap.

**Comment by crashmajstr** [comment]
Hi @madve2 
it should be fixed in the latest update 3.5.4 Give it a try and let me know. 
Best,
Josef

**Comment by madve2** [comment]
I can confirm it's fixed, thank you!
